resources/css/style.css

```diff
@@ -405,11 +405,159 @@ footer p {
 }
 
 
+/*-----------------------------------------*/
+/* SECTION DASHBOARD */
+/*-----------------------------------------*/
+
+
+.modal-dialog-dashboard {
+	max-width: 95%;
+}
+
+.modal-content-dashboard {
+	width: 100%;
+}
+
+.close-dashboard {
+	position: relative;
+	left: 100%;
+	cursor: pointer;
+}
+.cart-icon {
+	position: fixed;
+	padding: 15px 20px 15px 15px;
+	margin-top: 70px;
+	color: #fff;
+	cursor: pointer;
+	background-color: #648B79;
+	z-index: 100000;
+	font-size: 200%;
+}
+
+.cart-icon span {
+	font-size: 100%;
+	font-weight: 600;
+	color: #e67e22;
+}
+
+table {
+	margin: 2% 0;
+	width: 100%;
+	border: 1px solid #fff;
+	box-shadow: 5px 5px 20px #888888;
+}
+
+th {
+	width: 20%;
+	height: 7vh;
+	text-align: center;
+	background-color: #464242;
+	color: #e67e22;
+	font-size: 120%;
+	font-weight: 600;
+}
+
+td {
+	background-color: #f0f0f2;
+	text-align: center;
+	color: #e67e22;
+	font-weight: 300;
+	height: 5vh;
+	border-bottom: 1px solid #fff;
+	border-top: 1px solid #fff;
+}
+
+table span {
+	position: relative;
+	width: 50%;
+	float: left;
+	text-align: right;
+	height: 50%;
+	top: 30%;
```

Comment:
If it's position relative, wouldn't be better to use margin top or padding top? I'm not user what's the scenario here actually :)

Reply:
The scenario here was that i wanted the text in this span to perfectly align with the line between the 2 buttons(+ and -).
I modified it now and it looks better with just a padding-top. Thanks. :)